### ISSUE TITLE
Embedded

### ISSUE THREAD

**Issue by andrade94** [mpetroff/pannellum#273]
Is there a way for me to host the tour in my webpage that was created with the API and create an embedded link for me to embed it on another person's webpage?

**Comment by mpetroff** [comment]
Make it a separate page and use an `<iframe>`. This is how the [standalone version](https://github.com/mpetroff/pannellum/blob/master/src/standalone/pannellum.htm) works (as well as things like YouTube embeds).

**Comment by andrade94** [comment]
So I should use the standalone.js too? and the API things I have for my tour should be done in that html pannellum.htm?

**Comment by mpetroff** [comment]
No, you shouldn't use `standalone.js` is you're using the API. I just mentioned `pannellum.htm` as an example of using the Pannellum API in an `<iframe>`. The embed code would be something like this: `<iframe width="480" height="390" allowfullscreen style="border-style:none;" src="youriframe.html"></iframe>`

**Comment by strarsis** [comment]
@mpetroff: So using the iframe approach (standalone.html) is the recommended/best practice approach for putting a panorama on a web page?
For frameworks it is also possible to add the pannellum javascripts and the initialization directly (on a DIV element that should hold the pannellum viewer) - this approach may reduce issues with text leaking into search results.

**Comment by mpetroff** [comment]
It's the recommend approach if you want to allow other people to embed it, which is what this issue is about. For other use cases, it depends. The `<iframe>` approach is slightly simpler, while the API allows a lot more flexibility.
